kolibri/plugins/learn/assets/src/vue/card-grid/index.vue

```diff
@@ -1,7 +1,10 @@
 <template>
 
   <div>
-    <h3 v-if="header" class="header">{{header}}</h3>
+    <div class="header">
+      <h3 v-if="header">{{header}}</h3>
```

Comment:
I was being consistent with the existing implementation. ("Most Popular" and other expandable grids also use `h3` with the same styling.)

If we change one we'll need to change them all, so I'd recommend doing that as a separate task.